crates/forge/src/test_case_summary.rs

```diff
@@ -17,6 +17,7 @@ pub enum TestCaseSummary {
         msg: Option<String>,
         /// Arguments used in the test case run
         arguments: Vec<Felt252>,
+        runs: Option<u32>,
```

Comment:
Imo this being option is weird. Either it should be 1 (which is not complete as fuzzing can fail after one run) if not fuzzed or we should have `Option<FuzzingStatistics>``

Reply:
I don't understand what you mean

Reply:
@piotmag769 markdown cut the code snippet, fixed